requirements.txt

```diff
@@ -2,13 +2,13 @@ jupyterlab
 jupytext
 notebook
 matplotlib
-scikit-image==0.21.0
-jupyter-book==0.15.1
-pooch==1.7.0
-scikit-learn==1.2.2
-ipywidgets==8.0.7
-seaborn==0.12.2
-napari==0.4.18
+scikit-image==0.24.0
+jupyter-book==1.0.0
+pooch==1.8.2
+scikit-learn==1.5.0
+ipywidgets==8.1.3
+seaborn==0.13.2
+napari==0.4.19.post1
 pyqt5
 cookiecutter
 cellpose
```

Comment:
Considering that we probably won't be using the `04_deep_learning_segmentation.md` exercise, cellpose and its torch dependency seems very over-the-top. Let's remove it and add a comment to the top of the example in case someone really wants to try it? Or just remove the exercise altogether. What do you think? :thinking:

Reply:
@lagru I had s similar problem with cellpose and torch, if cellpose was imported first it didn't crash. When imported after torch it would seg fault.
I fixed by always setting cellpose as the first import.

Reply:
That's horrible! :sweat_smile: I heard that pytorch envs can be messy/problematic but never really had much exposure...